packages/genesys-spark-components/src/components/stable/gux-dismiss-button/gux-dismiss-button.scss

```diff
@@ -1,7 +1,28 @@
+:host {
+  // all these to be removed when token issues are fixed
```

Comment:
Should we create a JIRA ticket to track this and add a TODO?

Reply:
Waiting for Daragh on that. He said he'd have one likely by the end of the week if he can't get his fixes to work.